src/Components/ProvisioningWizard/steps/AccountCustomizations/aws.js

```diff
@@ -14,12 +14,13 @@ import { useWizardContext } from '../../../Common/WizardContext';
 import TemplatesSelect from '../../../TemplateSelect';
 
 const AccountCustomizationsAWS = ({ setStepValidated, image }) => {
-  const [{ chosenSource, chosenRegion, chosenInstanceType }, setWizardContext] = useWizardContext();
+  const [{ chosenSource, chosenRegion, chosenInstanceType, chosenNumOfInstances }, setWizardContext] = useWizardContext();
   const [validations, setValidation] = React.useState({
     sources: chosenSource ? 'success' : 'default',
     types: chosenInstanceType ? 'success' : 'default',
   });
   const templateFlag = useFlag('provisioning.aws.templates');
+  const [vCPUCount, setvCPUCount] = React.useState(chosenNumOfInstances);
```

Comment:
I'd do null default here, this default doesn't make much sense, we can do 0, if you're worried about handling special cases when doing math ops on this.